task_sdk/src/airflow/sdk/api/client.py

```diff
@@ -161,6 +162,15 @@ def set_rtif(self, id: uuid.UUID, body: dict[str, str]) -> dict[str, bool]:
         # decouple from the server response string
         return {"ok": True}
 
+    def get_previous_successful_dagrun(self, id: uuid.UUID) -> PrevSuccessfulDagRunResponse:
+        """
+        Get the previous successful dag run for a given task instance.
+
+        The data from it is used to get values for Task Context.
+        """
+        resp = self.client.get(f"task-instances/{id}/previous_successful_dagrun")
```

Comment:
Consistency of URL please 
```suggestion
        resp = self.client.get(f"task-instances/{id}/previous-successful-dagrun")
```